activemodel/test/cases/type/float_test.rb

```diff
@@ -21,9 +21,11 @@ def test_type_cast_float_from_invalid_string
       def test_changing_float
         type = Type::Float.new
 
-        assert type.changed?(5.0, 5.0, "5wibble")
+        assert type.changed?(5.0, 0, "wibble")
```

Comment:
Can you add `assert type.changed?(0, 0, "wibble")` too?
Since `type.changed?` always returns true if differ `old_value` and `new_value` regardless of the result of `non_numeric_string?`.